docs/openapi/components/schemas/common/TransformEvent.yml

```diff
@@ -0,0 +1,203 @@
+$linkedData:
+  term: TransformEvent
```

Comment:
If this is meant as steel-specific, should that be reflected in the definition? A `SteelTransformEvent` could be a type of a more generic `SupplyChainValueAdd` event.

Reply:
should be a generic one, not steel specific. enums for eventType have be removed.